CODE/VulnScan/tools/_vectorizer.py

```
@@ -0,0 +1,53 @@
+
+import os
+
+
+def load_data(data_paths: str | os.PathLike) -> list[str]:
+    data = []
+    if os.path.isdir(data_paths):
+        for root, _, files in os.walk(data_paths):
+            for file in files:
+                print("Loading File: ", file)
+                file_path = os.path.join(root, file)
+                with open(file_path, 'r', encoding='utf-8') as f:
```

Ruff (0.8.0)

Unnecessary open mode parameters

Remove open mode parameters

(UP015)